caput/containers/__init__.py

```diff
@@ -1,13 +1,62 @@
-"""caput.containers.
+"""A high-level in-memory data container format for :py:mod:`caput.pipeline`.
 
-A high-level in-memory data container format for `caput.pipeline`.
+Containers are built on top of :py:class:`~caput.memdata.BasicCont` and provide a structured way
+to hold multi-dimensional data arrays with named axes and dataset specifications.
+New containers should generally inherit from :py:class:`.ContainerBase` or
+:py:class:`.TableBase`.
 
-Submodules
-----------
-.. autosummary::
-   :toctree: _autosummary
+Examples
+--------
+Containers are defined by subclassing :py:class:`ContainerBase` and specifying
+the axes and dataset specifications:
```

Comment:
```suggestion
Containers are subclasses of :py:class:`ContainerBase`.  New containers are defined by specifying
the axes and datasets when subclassing:
```
The worry here being people might think that _only_ `ContainerBase` should be subclassed (instead of, say, subclassing `TableBase`).

Also remove a duplication of "specify/specification".